src/lib/edge-functions/internal.mjs

```diff
@@ -36,9 +36,16 @@ export const getInternalFunctions = async () => {
     const stats = await stat(path)
 
     if (!stats.isDirectory()) {
-      throw new Error('Path is not a directory')
+      throw new Error('Internal edge functions directory expected')
     }
+  } catch {
+    return {
+      functions: [],
+      path: null,
```

Comment:
If we wanted to be a bit more consistent, we could make `path: undefined` here and then in `src/lib/edge-functions/proxy.mjs` make the check:

```js
const hasEdgeFunctions = userFunctionsPath !== undefined || internalFunctionsPath !== undefined
```

Not a huge deal, though.